implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -53,6 +55,10 @@ pub struct CreateCommand {
     #[arg(display_order = 900, long, short)]
     pub foreground: bool,
 
+    /// Watch stdin for EOF
+    #[arg(display_order = 900, long = "watch-stdin", short)]
+    pub watch_stdin: bool,
```

Comment:
@hairyhum should this be the default behavior? Also, would it make more sense to rename it to `exit-on-eof`? (cc @phillyphil91)

Reply:
I don't think it should be default, but I like the `exit-on-eof` arg name.